lua/lvim/bootstrap.lua

```diff
@@ -64,15 +64,25 @@ function M:init(base_dir)
   self.lazy_install_dir = join_paths(self.pack_dir, "lazy", "opt", "lazy.nvim")
 
   ---@meta overridden to use LUNARVIM_CACHE_DIR instead, since a lot of plugins call this function internally
-  ---NOTE: changes to "data" are currently unstable, see #2507
   ---@diagnostic disable-next-line: duplicate-set-field
   vim.fn.stdpath = function(what)
     if what == "cache" then
       return _G.get_cache_dir()
+    elseif what == "config" then
```

Comment:
Don't forget to update the corresponding test, see https://github.com/LunarVim/LunarVim/pull/2540/files#diff-e02939ff80b05af32ad541b952d46570ab42ca3386e9eeab4bd4990955af4687